### PR TITLE
M2-609 Builder about activity

### PR DESCRIPTION
**Pull request by wildkiomi**
image

### CODE REVIEW

File: src/features/Builder/AboutApplet/AboutApplet.schema.ts [hunk 1]
```diff
@@ -4,16 +4,14 @@ import i18n from 'i18n';
...
-  const descriptionRequired = t('descriptionRequired');
-  const aboutAppletRequired = t('aboutAppletRequired');
+  const fieldRequired = t('fieldRequired');
 
   return yup
     .object({
-      name: yup.string().required(nameRequired).max(55),
-      description: yup.string().required(descriptionRequired).max(230),
+      name: yup.string().required(fieldRequired).max(55),
+      description: yup.string().required(fieldRequired).max(230),
```
Comment (IhorNesterenko99) [inline src/features/Builder/AboutApplet/AboutApplet.schema.ts:12]: Could you pls move these max to variables?

File: src/features/Builder/ActivityAbout/ActivityAbout.schema.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import { MAX_NAME_LENGTH } from 'consts';
+
+export const ActivityAboutSchema = () => {
+  const { t } = i18n;
+  const fieldRequired = t('fieldRequired');
+
+  return yup
+    .object({
+      activityName: yup.string().required(fieldRequired).max(MAX_NAME_LENGTH),
+      activityDescription: yup.string().required(fieldRequired).max(230),
```
Comment (IhorNesterenko99) [inline src/features/Builder/ActivityAbout/ActivityAbout.schema.ts:13]: Also here to variable

File: src/features/Builder/ActivityAbout/ActivityAbout.tsx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+              {...commonProps}
+              name="activityName"
+              maxLength={MAX_NAME_LENGTH}
+              label={t('activityName')}
+              sx={{ marginBottom: theme.spacing(4.4) }}
+            />
+            <InputController
+              {...commonProps}
+              name="activityDescription"
+              maxLength={230}
```
Comment (IhorNesterenko99) [inline src/features/Builder/ActivityAbout/ActivityAbout.tsx:70]: Also to variable

File: src/features/Builder/ActivityAbout/ActivityAbout.tsx [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+            <StyledUploadImg>
+              <StyledTitle>
+                {t('activityImg')}
+                <Tooltip tooltipTitle={mockedTooltipText}>
+                  <span>
+                    <StyledSvg id="more-info-outlined" />
+                  </span>
+                </Tooltip>
+              </StyledTitle>
+              <Uploader width={20} height={20} />
```
Comment (IhorNesterenko99) [inline src/features/Builder/ActivityAbout/ActivityAbout.tsx:87]: Maybe we also need to pass control inside uploaders?

File: src/features/Builder/ActivityAbout/ActivityAbout.const.ts [hunk 5]
```diff
@@ -0,0 +1,8 @@
+export const defaultValues = {
+  activityName: '',
```
Comment (IhorNesterenko99) [inline src/features/Builder/ActivityAbout/ActivityAbout.const.ts:2]: why don't we have images here?

File: src/features/Builder/ActivityAbout/ActivityAbout.tsx [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+    defaultValues,
+    mode: 'onChange',
+  });
+
+  const commonProps = {
+    control,
+    fullWidth: true,
+  };
+
+  return (
```
Comment (kdanilenka32) [inline src/features/Builder/ActivityAbout/ActivityAbout.tsx:54]: better to move repeating logic that in AboutApplet

File: src/features/Builder/ActivityAbout/ActivityAbout.tsx (hunk 4 above)
Comment (kdanilenka32) [reply src/features/Builder/ActivityAbout/ActivityAbout.tsx:87]: > Maybe we also need to pass control inside uploaders?

no need yet, I am working on crop, I will add if needed

**Review by IhorNesterenko99 — APPROVED** [review]

**Review by nmialik — APPROVED** [review]

**Pull request merged**